### ISSUE TITLE
npm release

### ISSUE THREAD

**Issue by SgtPooki** [dutu/rate-limiter#3]
any plans to release this package to npm?

**Comment by dutu** [comment]
I'm not planning to release it to npm, however it can be installed from github by using `git+https://github.com/dutu/rate-limiter.git`

**Comment by dutu** [comment]
I didn't see a need to release it to npm, Is there an inconvenient fetching it from github directly?

**Comment by SgtPooki** [comment]
definitely. `npm install https://github.com/dutu/rate-limiter.git` is much more difficult than `npm install @dutu/rate-limiter`
